content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -161,6 +165,10 @@ scroll to the :guilabel:`Colombian Electronic Invoicing` section. Then, follow t
      To activate it, disable both the :guilabel:`Test environment` and the :guilabel:`Activate the
      certification process` checkboxes.
 
+.. warning::
+   Do not activate **DIAN Demo Mode** in a production database, this mode is only intended for testing
+   databases.
+
```

Comment:
Slight RST syntax adjustments and indenting to reside in the production env bullet
```suggestion
     .. warning::
        Do not activate :guilabel:`DIAN Demo Mode` on a production database, this mode is only
        intended for testing databases.
```